addon/initializers/override-built-in-macros.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable ember/new-module-imports */
+import Ember from 'ember';
+import computedStore from '../-computed-store';
+
+const _computed = Ember.computed;
+
+export function initialize() {
```

Comment:
@kellyselden this doesn't work. When `raw` (or `Ember.computed`) is used in a class definition, it is evaluated at module load time, which happens before initializers execute, so the instrumentation is not in place.